tests/renderer/app.test.js

```diff
@@ -360,31 +365,37 @@ describe('Display recently executed InVEST jobs on Home tab', () => {
 describe('InVEST global settings: dialog interactions', () => {
   const nWorkersLabelText = 'Taskgraph n_workers parameter';
   const loggingLabelText = 'Logging threshold';
+  const languageLabelText = 'Language';
 
   beforeEach(async () => {
     getInvestModelNames.mockResolvedValue({});
+    ipcRenderer.invoke.mockResolvedValue({});
   });
 
   test('Invest settings save on change', async () => {
     const nWorkersLabel = 'Threaded task management (0)';
     const nWorkersValue = '0';
     const loggingLevel = 'DEBUG';
+    const languageValue = 'es';
 
     const {
-      getByText, getByRole, getByLabelText, findByRole
+      getByText, getByRole, getByLabelText, findByRole,
     } = render(
       <App />
     );
 
     userEvent.click(await findByRole('button', { name: 'settings' }));
     const nWorkersInput = getByLabelText(nWorkersLabelText, { exact: false });
     const loggingInput = getByLabelText(loggingLabelText, { exact: false });
+    const languageInput = getByLabelText(languageLabelText, { exact: false });
 
     userEvent.selectOptions(nWorkersInput, [getByText(nWorkersLabel)]);
     userEvent.selectOptions(loggingInput, [loggingLevel]);
+    userEvent.selectOptions(languageInput, [languageValue]);
     await waitFor(() => {
       expect(nWorkersInput).toHaveValue(nWorkersValue);
       expect(loggingInput).toHaveValue(loggingLevel);
+      expect(languageInput).toHaveValue(languageValue);
```

Comment:
I commented this elsewhere, I think it's best to change this to the `selectOptions` and `expect` for each input one at a time, instead of the batches of actions followed by batches of assertions.